File: Data-Types/1. Character/1. ASCII Code/ASCII Code.cs
using System;

namespace ASCII
{
    class ASCII
    {
        static void Main(string[] args)
        {
            char ch;
            Console.Write("\n Enter a character : ");
            ch = char.Parse(Console.ReadLine());

            int n=ch;
            Console.Write("\n ASCII code of '" + ch + "' is " + n + ".\n");
            Console.ReadKey();
        }
    }
}

File: Data-Types/1. Character/1. ASCII Code/ASCII_Code.java
import java.util.Scanner;

class ASCII
{
    public static void main(String args[])
    {
        Scanner s=new Scanner(System.in);
        
        char ch;        
        System.out.print("\n Enter a character : ");
        ch=s.next().charAt(0);
        
        int n=ch;
        System.out.print("\n ASCII code of '"+ch+"' is "+n+".\n");
    }
}